.github/workflows/manage-visual-tests.yml

```diff
@@ -0,0 +1,297 @@
+  BRANCH_NAME: ${{ github.event.pull_request.head.ref }}
+  BASE_COMMIT_SHA: ${{ github.event.pull_request.base.sha }}
+  HEAD_COMMIT_SHA: ${{ github.event.pull_request.head.sha }}
+  PR_NUMBER: ${{ github.event.pull_request.number }}
+  WORKFLOW_ID: ${{ github.run_id }}
+  REPOSITORY_NAME: ${{ github.event.repository.name }}
+  REPOSITORY_OWNER: ${{ github.repository_owner }}
+
+jobs:
+  split-into-groups:
+    if: github.event.action != 'closed' && contains(github.event.pull_request.labels.*.name, 'run-visual-tests')
+    name: Split into groups
+    runs-on: ubuntu-latest
+    outputs:
+      stories-groups: ${{ steps.get-stories-groups.outputs.stories-groups }}
+    steps:
+      - name: '[Prepare] Checkout'
+        uses: actions/checkout@v4
+
+      - name: '[Prepare] Setup Node.js'
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: 'package.json'
+          cache: 'npm'
+
+      - name: '[Prepare] Install dependencies'
+        run: npm ci
+
+      # Expected outputs
+      # - stories-groups=Array<`stories-group-{number}`>
+      - name: '[Run] Get stories groups'
+        id: get-stories-groups
+        run: node ./.github/scripts/get-stories-groups.js
+
+  check-for-expectation-update:
+    if: github.event.action != 'closed' && contains(github.event.pull_request.labels.*.name, 'run-visual-tests')
+    name: Check for last expectation update
+    runs-on: ubuntu-latest
+    outputs:
+      should-update-expectation: ${{ steps.check-for-expectation-update.outputs.should-update-expectation }}
+      last-expectation-update-utc: ${{ steps.check-for-expectation-update.outputs.last-expectation-update-utc }}
+      last-expectation-update-local: ${{ steps.check-for-expectation-update.outputs.last-expectation-update-local }}
+    steps:
+      - name: '[Prepare] Checkout'
+        uses: actions/checkout@v4
+
+      - name: '[Prepare] Setup Node.js'
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: 'package.json'
+          cache: 'npm'
+
+      - name: '[Prepare] Install dependencies'
+        run: npm ci
+
+      # Expected outputs:
+      # - should-update-expectation: boolean - true if expectation should be updated (base commit has changed or no previous report was found)
+      # - last-expectation-update-utc: string - date as string representing the last update of the expectation screenshots (UTC)
+      # - last-expectation-update-local: string - date as string representing the last update of the expectation screenshots (Local)
+      - name: '[Run] Check for last expectation update'
+        id: check-for-expectation-update
+        run: node ./tasks/visual-tests/manage-visual-tests-data.js check-for-expectation-update
+        env:
+          VISUAL_TESTS_CELLAR_KEY_ID: ${{ secrets.VISUAL_TESTS_CELLAR_KEY_ID }}
+          VISUAL_TESTS_CELLAR_SECRET_KEY: ${{ secrets.VISUAL_TESTS_CELLAR_SECRET_KEY }}
```

Comment:
it's fairly easy to make a node script output stuff for other steps / jobs in the workflow but the drawback is that these outputs are undocumented.
The only way I found to mitigate that was to add comments :shrug: